modin/pandas/test/test_series.py

```diff
@@ -3617,3 +3617,32 @@ def test_str_isdecimal(data):
     else:
         modin_result = modin_series.str.isdecimal()
         df_equals(modin_result, pandas_result)
+
+
+@pytest.mark.parametrize("data", test_string_data_values, ids=test_string_data_keys)
```

Comment:
Why were these tests added for `nunique`?

Reply:
After is_unique implementation, two functions became uncovered by testing, so these new tests fix this problem.

Reply:
👍